Client/Frontend/Library/Bookmarks/BookmarksPanel.swift

```diff
@@ -107,10 +107,13 @@ class BookmarksPanel: SiteTableViewController, LibraryPanel, CanRemoveQuickActio
     }
 
     override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
-        if tableView.isEditing {
-            self.tableView.setEditing(false, animated: true)
-        }
         super.viewWillTransition(to: size, with: coordinator)
+
+        coordinator.animate(alongsideTransition: nil) { _ in
+            if self.state == .bookmarks(state: .inFolderEditMode) {
+                self.tableView.setEditing(true, animated: true)
+            }
+        }
```

Comment:
I test it and it does fix the issue my only suggestion is to do it outside of the animation closure because with the animation you can see the tableview state changes
```suggestion

          if state == .bookmarks(state: .inFolderEditMode) {
              tableView.setEditing(true, animated: true)
          }
```